dev/plugins/hu.elte.txtuml.xtxtuml.ui/src/hu/elte/txtuml/xtxtuml/ui/contentassist/XtxtUMLReferenceProposalCreator.xtend

```diff
@@ -274,16 +271,12 @@ class XtxtUMLReferenceProposalCreator extends XbaseReferenceProposalCreator {
 			// be difficult to achieve starting from a plain JvmType
 			proposedObj instanceof JvmDeclaredType && (proposedObj as JvmDeclaredType).superTypes.exists [
 				val typeRef = toLightweightTypeReference;
-				typeRef.isSubtypeOf(DataType) || typeRef.isInterface && typeRef.isSubtypeOf(ExternalType) || typeRef.isSubtypeOf(ModelEnum) ||
```

Comment:
Done in d6c9c47, see the full commit message and my [comment](https://github.com/ELTE-Soft/txtUML/issues/507#issuecomment-314433145) for details.